### PR TITLE
Update windows packaging to support containerd

### PR DESCRIPTION
**Pull request by lmm**
## Description

Update windows packaging to configure kubelet for containerd if containerd service is running.

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
Windows packaging for containerd support
```

### CODE REVIEW

File: windows-packaging/CalicoWindows/libs/calico/calico.psm1 [hunk 1]
```diff
@@ -413,6 +413,47 @@ function Set-MetaDataServerRoute($mgmtIP)
...
+    # Don't use get-wmiobject since that is not available in Powershell 7.
+    return Get-CimInstance -Query "SELECT * from Win32_Service WHERE name = 'containerd'"
+}
+
+function getContainerdConfig()
+{
+    if (!(Get-InstalledModule PsIni))
+    {
+        Write-Host "Dependency PsIni not installed, installing..."
+        Install-module PsIni -Confirm:$False -Force
```
Comment (song-jiang) [inline windows-packaging/CalicoWindows/libs/calico/calico.psm1:439]: I'm not sure what is the security implication of importing a third party module. Can we do it without this module?

**Review by song-jiang — COMMENTED** [review]
One comment

File: windows-packaging/CalicoWindows/libs/calico/calico.psm1 (hunk 1 above)
Comment (lmm) [reply windows-packaging/CalicoWindows/libs/calico/calico.psm1]: We discussed this and agree to remove this dependency and instead use a sensible default.

**Review by song-jiang — APPROVED** [review]
LGTM

**Pull request merged**